modules/RTC/DataChannels.js

```diff
@@ -13,6 +13,7 @@ function DataChannels(peerConnection, emitter) {
     peerConnection.ondatachannel = this.onDataChannel.bind(this);
     this.eventEmitter = emitter;
 
+    this._lastNEndpoints = null;
```

Comment:
It feels like it's convenient to have this variable here, but like Saul said it would be better to keep DataChannels a  message bus. One idea would be to move `emit(RTCEvents.LASTN_ENDPOINT_CHANGED...` to RTC. I guess DataChannels are using RTC's event emitter anyway.